### DISCUSSION TITLE
Use of the the 15min forecasts

### DISCUSSION THREAD

**Discussion by jortaca** [open-meteo/open-meteo#1033] [Q&A]
I have been testing the 15min forecast and I am not sure if I am doing something wrong, as it seems to be always giving interpolated data when trying to get the **wind** parameters.

When not selecting a model, for a **Spanish location**, it seems to use Icon, which is not my best choice, as I am mostly interested in wind and gusts data and Arome is the one which excels in this area. Also, Icon data seems interpolated for these parameters.

So I have tried to directly select the model using:
https://api.open-meteo.com/v1/forecast?latitude=40.7254&longitude=0.8399&minutely_15=wind_speed_10m,wind_direction_10m,wind_gusts_10m&models=meteofrance_arome_france_hd

In the documentation says that Arome doesn't have the Wind Gusts 10m for the 15min forecast, but I am getting data although it seems interpolated. The Wind 10m also looks like it is interpolated. That's why I am asking. In the documentation, it says that the only model with all these parameters is HHRR:

![Captura de pantalla 2024-09-23 a las 10 16 01](https://github.com/user-attachments/assets/f264c7fe-6646-4159-ae3d-2d3619ecac9d)

Is this right?
But is the HHRR with 11 km range por Spain?

For my app, I also need it to get it for many locations around the globe. What I do with the 1 hour models is that if the data is null, I get it from other model with less resolution. But the 15 min forecast seems to always give interpolated data if it doesn't have it, so I don't have the way to know which one to choose. Any ideas?

**Comment by patrick-zippenfenig** [comment]
Hi, 15 minutely data is currently only applied for `best_match`, `arome_seamless` and `meteofrance_seamless`. The regular AROME and AROME-HD are only returning 1-hourly data. I will change it that `arome_france` and `arome_france_hd` will also use 15 minutely data.

See: https://github.com/open-meteo/open-meteo/issues/1034

**Reply by jortaca** [reply]
The problem for me with the best_match is that gives me the Icon interpolated data for Spain. So that's why I am trying to use Arome directy. 
Has the arome_seamless or the meteofrance_seamless the wind_gusts parameter?
Does the arome_seamless include the arome_france_hd data? (If it does, it is not needed the _hd directy, isn't it?)

**Reply by patrick-zippenfenig** [reply]
Yes `arome_seamless ` includes the normal AROME and AROME HD. Only AROME HD has wind gust data in 15min

**Reply by jortaca** [reply]
Thank you!
I have changed it to arome_seamless and results are promising.

**Reply by jortaca** [reply]
Hi again, Patrick. I have just one more question (and apologies for being so persistent about this). I'm currently using the meteofrance_seamless model (is it the same as arome_seamless?), and it's providing wind gust data. However, the gust data seems interpolated. While the wind data shows peaks and drops, the gusts remain relatively flat. In some instances, the gusts are even lower than the wind speed.

![Captura de pantalla 2024-09-30 a las 14 33 59](https://github.com/user-attachments/assets/0a3bead4-a425-42aa-ba6e-b0cab87ac207)

https://open-meteo.com/en/docs#latitude=46.06572491967881&longitude=-1.0935745264578147&minutely_15=wind_speed_10m,wind_gusts_10m&hourly=&wind_speed_unit=kn&forecast_days=1&models=meteofrance_seamless

Could this be related to the wind gust issue you previously addressed in the normal Arome HD model? (https://github.com/open-meteo/open-meteo/issues/888)

Thanks again!

**Reply by patrick-zippenfenig** [reply]
`meteofrance_seamless` uses AROME (France 0.025° + 0.01°) and ARPEGE  (Global+EU). `arome_seamless` only uses AROME models.

You are right. There is are no 15min gusts in any AROME domain: https://openmeteo.s3.amazonaws.com/index.html#data/meteofrance_arome_france0025_15min/

**Reply by jortaca** [reply]
While looking for more info about this, I have found the Meteociel Arome PI webpage, where it features 15min. Arome PI 10m. Gusts and Gusts Max.Ech (don't know what that means) data. (https://www.meteociel.fr/modeles/aromepi.php?ech=3&mode=11&map=710)

Could that be used to add it as Arome 15min windgusts data?

**Reply by patrick-zippenfenig** [reply]
It looks like wind gusts are available again through the MeteoFrance API. I just added it. Data should be available in the next 2-3 hours. https://github.com/open-meteo/open-meteo/pull/1050

**Reply by jortaca** [reply]
Great!!!!
